free-courses-en.md

```diff
@@ -77,7 +77,7 @@
 
 ### Bash / Shell
 
-* [Bash tutorial](http://gdrcorelec.ups-tlse.fr/files/bash.pdf) (PDF)
+
```

Comment:
I will make another pull request to remove the bash book from the online course section and  put it in the book section. Please check it out :)